frontend/src/components/CourseModal/Header/ControlsRow.module.css

```
@@ -47,24 +49,27 @@
+.shareDropdown svg {
   color: var(--color-primary);
 }
 
 .moreDropdown:hover svg,
-.shareButton:hover svg {
+.shareDropdown:hover svg {
   color: var(--color-primary-hover);
 }
 
-.moreDropdown :global(.dropdown-toggle) {
+.moreDropdown :global(.dropdown-toggle),
```

Stylelint (17.2.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)